CI/results/report-results.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Exit positively if we are on a PR
+if [[ "$TRAVIS_EVENT_TYPE" == "pull_request" ]];
+then
+  exit 0
```

Comment:
@lauraquellmalz 
I think this would result in a always passed result, even when the unittests fail, that's why we have this construct here: https://github.com/ILIAS-eLearning/ILIAS/blob/release_6/CI/PHPUnit/run_tests_with_reporting.sh#L88 so if the unittest fail, we need another exit code, instead of 0. So the problem is not that simple. ;)

Reply:
Oooooooh. I see ...